tests/e2e/actor_metadata/actor_metadata_test.go

```diff
@@ -0,0 +1,280 @@
+	secondsToCheckReminderResult = 45                                   // How much time to wait to make sure the result is in logs.
+	maxNumPartitions             = 6                                    // Maximum number of partitions.
+	actorName                    = "testactormetadata"                  // Actor name
+	actorInvokeURLFormat         = "%s/test/" + actorName + "/%s/%s/%s" // URL to invoke a Dapr's actor method in test app.
+	actorlogsURLFormat           = "%s/test/logs"                       // URL to fetch logs from test app.
+	envURLFormat                 = "%s/test/env/%s"                     // URL to fetch or set env var from test app.
+	shutdownSidecarURLFormat     = "%s/test/shutdownsidecar"            // URL to shutdown sidecar only.
+)
+
+// represents a response for the APIs in this app.
+type actorLogEntry struct {
+	Action         string `json:"action,omitempty"`
+	ActorType      string `json:"actorType,omitempty"`
+	ActorID        string `json:"actorId,omitempty"`
+	StartTimestamp int    `json:"startTimestamp,omitempty"`
+	EndTimestamp   int    `json:"endTimestamp,omitempty"`
+}
+
+type actorReminder struct {
+	Data     string `json:"data,omitempty"`
+	DueTime  string `json:"dueTime,omitempty"`
+	Period   string `json:"period,omitempty"`
+	TTL      string `json:"ttl,omitempty"`
+	Callback string `json:"callback,omitempty"`
+}
+
+// renameReminderRequest is the request object for rename a reminder.
+type renameReminderRequest struct {
+	OldName   string
+	ActorType string
+	ActorID   string
+	NewName   string
+}
+
+type reminderResponse struct {
+	ActorID        string      `json:"actorID,omitempty"`
+	ActorType      string      `json:"actorType,omitempty"`
+	Name           string      `json:"name,omitempty"`
+	Data           interface{} `json:"data"`
+	Period         string      `json:"period"`
+	DueTime        string      `json:"dueTime"`
+	RegisteredTime string      `json:"registeredTime,omitempty"`
+	ExpirationTime string      `json:"expirationTime,omitempty"`
+}
+
+func parseLogEntries(resp []byte) []actorLogEntry {
+	logEntries := []actorLogEntry{}
+	err := json.Unmarshal(resp, &logEntries)
+	if err != nil {
+		return nil
+	}
+
+	return logEntries
+}
+
+func countActorAction(resp []byte, actorID string, action string) int {
+	count := 0
+	logEntries := parseLogEntries(resp)
+	for _, logEntry := range logEntries {
+		if (logEntry.ActorID == actorID) && (logEntry.Action == action) {
+			count++
+		}
+	}
+
+	return count
+}
+
+var tr *runner.TestRunner
+
+func TestMain(m *testing.M) {
+	utils.SetupLogs("actor_metadata")
+	utils.InitHTTPClient(false)
+
+	// These apps will be deployed before starting actual test
+	// and will be cleaned up after all tests are finished automatically
+	testApps := []kube.AppDescription{
+		{
+			AppName:        appNameOne,
+			DaprEnabled:    true,
+			ImageName:      "e2e-actorfeatures",
+			SidecarImage:   "daprio/daprd:1.9.6",
```

Comment:
Bit of a nit, please use 
```suggestion
			SidecarImage:   "ghcr.io/dapr/daprd:1.9.6",
```

Since Docker Hub is rate-limited

Reply:
Yes, great point.